src/Init/Tactics.lean

```diff
@@ -250,7 +250,7 @@ macro:1 x:tactic tk:" <;> " y:tactic:2 : tactic => `(tactic|
     all_goals $y:tactic)
 
 /-- `eq_refl` is equivalent to `exact rfl`, but has a few optimizations. -/
-syntax (name := refl) "eq_refl" : tactic
+syntax (name := eqRefl) "eq_refl" : tactic
```

Comment:
This is fine, but a more radical suggestion is to simply remove it. It is not used in Lean/Std/Mathlib currently.

Reply:
Besides as one of the implementations of `rfl`, it seems:
```
macro "rfl" : tactic => `(tactic| eq_refl)
```